code/modules/games/cah.dm

```diff
@@ -0,0 +1,262 @@
+// This is a parody of Cards Against Humanity (https://en.wikipedia.org/wiki/Cards_Against_Humanity)
+// which is licensed under CC BY-NC-SA 2.0, the full text of which can be found at the following URL:
+// https://creativecommons.org/licenses/by-nc-sa/2.0/legalcode
+
+//Ported from Polaris;
+//To be rewritten to be made more 'in-universe' and less 'meta-SS13 references', as amusing the latter can be.
+//Pretty sure people in-universe wouldn't exactly see things the same way we SS13 players do -- Mwah
+
+/obj/item/deck/cah
+	name = "\improper CAG deck (white)"
+	desc = "The ever-popular Cards Against The Galaxy word game. This is the white deck."
+	icon_state = "cag_white"
+	var/blanks = 5
+
+/obj/item/deck/cah/black
+	name = "\improper CAG deck (black)"
+	desc = "The ever-popular Cards Against The Galaxy word game. This is the black deck."
+	icon_state = "cag_black"
+	blanks = 0
+
+/obj/item/deck/cah/New()
+	..()
+	var/datum/playingcard/P
+	for(var/cardtext in card_text_list)
+		P = new()
+		P.name = "[cardtext]"
+		P.card_icon = "[icon_state]_card"
+		P.back_icon = "[icon_state]_card_back"
+		cards += P
+	if(!blanks)
+		return
+	for(var/x=1 to blanks)
+		P = new()
+		P.name = "Blank Card"
+		P.card_icon = "[icon_state]_card_back"
+		P.back_icon = "[icon_state]_card_back"
+
+// Black cards.
+/obj/item/deck/cah/black/card_text_list = list(
+	"Why am I itchy?",
+	"Today, police shot ____.",
+	"Our local chaplain is worshipping _____.",
+	"My grandparents ordered a crate full of _____ from the catalogue.",
+	"The FIB was called due to ______.",
+	"Alert! The Sol Navy has armed themselves with _____.",
+	"Order received: ________ is your master.",
+	"Order received: ________ is the enemy.",
+	"_____ blew a hole in the wall.",
+	"Today, researchers found an artefact that made people ____ and ____. (Pick 2)",
+	"There was a rap battle between ____ and ____. (Pick 2)",
+	"Caution, ______ have been detected in collision course with the planet.",
+	"Today's kitchen menu includes _______.",
+	"What did space pirates want when they attacked our home?",
+	"I think our boss is insane. He just demanded ______ in his office.",
+	"Fuckin' scientists, they just turned the entire population of Scottsdan into _______ .",
+	"Fuckin' activists, they just converted Mendell City into ______ .",
+	"What's my fetish?",
+	"Next up on TCFL's Got Talent: ________ .",
```

Comment:
Using TCFL makes no sense here, replace with Tau Ceti or Biesel.

Reply:
I beg to differ.
It's a deliberate joke in a joke card game. It's precisely why it isn't so literal that it provides the context for an imaginable scenario by which the supposed Card Czar would base their scale upon. Replacing it with Tau Ceti or Biesel can work, but I figured it would be too literal—not to mention far too wide a cast for any actual TV show.

Reply:
Because it is a joke with the tv show: Britain's Got Talent. Tau Ceti/Biesel is the nation, so for the joke to make more sense, it should still be a country in our setting.

Reply:
I daresay,
Like I get it and I can see that point. I daresay though that there are more unofficial versions in which organisations also host a little 'Got Talent' thing, but I suppose that would go back to saying it's just a joke of the TV shows.
Now to be fair, I don't actually know what people think of the TCFL; I don't know if they think they're just the redshirts of the galaxy, or that the TCFL is just an expendable force a la the French Foreign Legion—but it's on those grounds from which I made that choice upon, rather than some sort of planet-wide TV show (which doesn't OFFICIALLY exist in the current lore as far as I'm aware of, if we're going by that anything not in the lore should be removed).

In that regard, could you see it make sense there? I'm not trying to go for the 'literal word-for-word translation' of parodies in THIS particular instance here where I'm making something like Carp Ahoy cookies based on Chips Ahoy cookie, etc.